src/main/java/ch/fmi/ContourPolygonBitmaskToText.java

```diff
@@ -0,0 +1,69 @@
+/*-
+ * #%L
+ * Plug-in to convert a 2D binary to a polygon geometry serialized to
+ * well-known text (wkt).
+ * %%
+ * @author Charismatic Claire
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package ch.fmi;
+
+import net.imagej.ops.OpService;
+import net.imagej.ops.Ops;
+import net.imglib2.img.Img;
+import net.imglib2.roi.geom.real.Polygon2D;
+import net.imglib2.type.logic.BitType;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Polygon;
+import org.scijava.ItemIO;
+import org.scijava.command.Command;
+import org.scijava.command.ContextCommand;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+@Plugin(type = Command.class, headless = true, menuPath = "FMI>2D Binary Mask To Text")
```

Comment:
I might take the opportunity and refine the menu structure a bit. Currently, all plugins from this repository are listed under a single *FMI* menu. It would make sense to create sub-menus for working with shapes, tracks, segmentations etc.

I will probably work on this before the next release, but if you have ideas or comments, I'm open for suggestions 😉

Reply:
I think I'll leave that to you since I only know what 3 plug-ins are doing :-)